tests/datetime_test.py

```diff
@@ -129,17 +136,37 @@ def test_create_datetime64_column_from_str():
     assert expr.values.astype(np.datetime64).tolist() == expr.astype('datetime64').tolist()
     assert expr.values.astype('datetime64[ns]').tolist() == expr.astype('datetime64[ns]').tolist()
 
+
 def test_create_str_column_from_datetime64():
     date = np.array([np.datetime64('2009-10-12T03:31:00'),
-                np.datetime64('2016-02-11T10:17:34'),
-                np.datetime64('2015-11-12T11:34:22'),
-                np.datetime64('2003-03-03T00:33:15'),
-                np.datetime64('2014-07-23T15:08:05'),
-                np.datetime64('2011-01-01T07:02:01')], dtype='<M8[ns]')
+                     np.datetime64('2016-02-11T10:17:34'),
+                     np.datetime64('2015-11-12T11:34:22'),
+                     np.datetime64('2003-03-03T00:33:15'),
+                     np.datetime64('2014-07-23T15:08:05'),
+                     np.datetime64('2011-01-01T07:02:01')], dtype='<M8[ns]')
 
     df = vaex.from_arrays(date=date)
     pandas_df = df.to_pandas_df()
 
     date_format = "%Y/%m/%d"
 
-    assert df.date.dt.strftime(date_format).values.tolist() == pandas_df.date.dt.strftime(date_format).values.tolist()
\ No newline at end of file
+    assert df.date.dt.strftime(date_format).values.tolist() == pandas_df.date.dt.strftime(date_format).values.tolist()
+
+
+def test_datetime_nat_minmax_consistency():
+    pandas_df = pd.DataFrame({'t': [pd.NaT,
+                                    datetime.datetime(2019, 1, 1),
+                                    datetime.datetime(2019, 2, 1),
+                                    datetime.datetime(2019, 4, 1),
+                                    datetime.datetime(2019, 5, 1)]})
+    df = vaex.from_pandas(pandas_df)
+
+    t_min = np.datetime64(df.t.min())
```

Comment:
This is not consistent, since we only ignore NaN, which is a special float number, NaT is kind of a cornercase (I wouldn't even know what 'value' is it actually, or where it is defined. I think this should be `df.t.dropna().min()`

Reply:
Agreed.

Reply:
should we close this PR?

Reply:
I  wonder if we can catch / intercept the pd.NaT and mask it, something like that.
Otherwise, I guess we can close it.

Reply:
I think we need to move to Arrow for datetime in the future, and then it will be masked values, till that time we need to live with it I think.